src/python-frontend/python_language.cpp

```diff
@@ -0,0 +1,81 @@
+#include "python-frontend/python_language.h"
+#include "python-frontend/python_converter.h"
+#include "util/message.h"
+
+#include <cstdlib>
+
+#include <boost/filesystem.hpp>
+#include <boost/process.hpp>
+
+namespace bp = boost::process;
+namespace fs = boost::filesystem;
+
+languaget *new_python_language()
+{
+  return new python_languaget;
+}
+
+bool python_languaget::parse(const std::string &path)
+{
+  log_debug("python", "Parsing: {}", path);
+
+  fs::path script(path);
+  if(!fs::exists(script))
+    return true;
+
+  fs::path esbmc_binary_path = fs::read_symlink("/proc/self/exe").parent_path();
+  fs::path python_script_path = esbmc_binary_path / "../resources/astgen.py";
```

Comment:
This will not work on native Windows. We could either hide it behind a #define, or do what we do with other objects distributed with ESBMC: bundle it in the binary and extract it during runtime. See the `flail.py` tool for bundling, also there is the CMake function `mangle` defined in scripts/cmake/Utils.cmake. For extracting I would point to src/c2goto/cprover_libc_sources.cpp where the standard headers are extracted when translating C sources.

Reply:
Nice. I will check.

Reply:
I opted for bundling astgen.py. Thanks for this information.